modules/menu/react/README.md

```diff
@@ -229,3 +230,8 @@ Default: `false`
 > Allow you to disable a menu item so it is not clickable.
 
 Default: `false`
+
+#### `canSkipClose: boolean`
```

Comment:
I didn't see a blurb about naming conventions on boolean variables. I see precedence for `hasX` and `isX`. Should this be named `shouldSkipClose` or maybe `shouldClose` defaulted to `true` to avoid the negative?

Reply:
Yep, I agree with @NicholasBoll, let's go with `shouldClose` that defaults to true 👍

Reply:
Done. Thanks for the feedback!